migrate/migrations/20230616110941-add-alerts-feedback.sql

```diff
@@ -0,0 +1,9 @@
+-- +migrate Up
+CREATE TABLE alert_feedback
+(
+    alert_id BIGINT PRIMARY KEY REFERENCES alerts (id) ON DELETE CASCADE,
+    noise_reason TEXT
```

Comment:
Should we make this not null?

It's not immediately obvious, but we're actually handling three different empty states in code:
- no rows
- noise_reason is `null`
- noise_reason is `""`

As written in this PR, only no rows and `""` are possible. If we need to migrate this in the future, it may be good to enforce consistency, plus getting rid of the `null` lets us use a Go `string` instead of `sql.NullString` that we aren't using.